src/lib/style/Portal.scss

```diff
@@ -813,7 +827,7 @@ checkmark styling end
   display: none;
 }
 
-.SRC-paddingTopNoMargin {
+div.col-xs-2.SRC-paddingTopNoMargin {
```

Comment:
Instead of targeting the grid layout class, perhaps add your own new class (in case the layout changes)?